.php-cs-fixer.php

```diff
@@ -18,51 +18,51 @@
 file that was distributed with this source code.
 EOF;
 
-return PhpCsFixer\Config::create()
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
+    ->in(['src','tests'])
+;
+
+$config = new PhpCsFixer\Config();
+$config->setUsingCache(true)
     ->setRiskyAllowed(true)
-    ->setRules([
-        'array_indentation' => true,
-        'array_syntax' => ['syntax' => 'short'],
-        'binary_operator_spaces' => ['align_double_arrow' => true, 'align_equals' => true],
-        'blank_line_after_namespace' => true,
-        'blank_line_after_opening_tag' => true,
-        'declare_strict_types' => true,
-        'header_comment' => ['header' => $header],
-        'indentation_type' => true,
-        'linebreak_after_opening_tag' => true,
-        'lowercase_constants' => true,
-        'lowercase_keywords' => true,
-        'method_separation' => true,
-        'native_function_invocation' => ['include' => ['@compiler_optimized']],
-        'no_alias_functions' => true,
-        'no_closing_tag' => true,
-        'no_extra_consecutive_blank_lines' => [
-            'tokens' => [
-                'break',
-                'continue',
-                'curly_brace_block',
-                'extra',
-                'parenthesis_brace_block',
-                'return',
-                'square_brace_block',
-                'throw',
-                'use',
-            ],
+    ->setFinder($finder);
+return $config->setRules([
+    'array_indentation' => true,
+    'array_syntax' => ['syntax' => 'short'],
+    'binary_operator_spaces' => ['default' => 'align', 'operators' => ['=>' => 'align', '=' => 'align_single_space']],
+    'blank_line_after_namespace' => true,
+    'blank_line_after_opening_tag' => true,
+    'declare_strict_types' => true,
+    'header_comment' => ['header' => $header],
+    'indentation_type' => true,
+    'linebreak_after_opening_tag' => true,
+    'constant_case' => true,
+    'lowercase_keywords' => true,
+    'native_function_invocation' => ['include' => ['@compiler_optimized']],
+    'no_alias_functions' => true,
+    'no_closing_tag' => true,
+    'no_extra_blank_lines' => [
+        'tokens' => [
+            'break',
+            'continue',
+            'curly_brace_block',
+            'extra',
+            'parenthesis_brace_block',
+            'return',
+            'square_brace_block',
+            'throw',
+            'use',
         ],
-        'no_short_echo_tag' => true,
-        'no_useless_else' => true,
-        'no_unused_imports' => true,
-        'no_useless_return' => true,
-        'ordered_imports' => true,
-        'phpdoc_add_missing_param_annotation' => true,
-        'phpdoc_order' => true,
-        'semicolon_after_instruction' => true,
-        'visibility_required' => true,
-    ])
-    ->setUsingCache(true)
-    ->setFinder(
-        PhpCsFixer\Finder::create()
-            ->in(__DIR__)
-            ->in(['src','tests'])
-    )
+    ],
+    'echo_tag_syntax' => true,
```

Comment:
looks like the expected value is an array https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.19.1/doc/rules/php_tag/echo_tag_syntax.rst